File: 프로그래머스/0/181886. 5명씩/5명씩.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> names) 
{
    vector<string> v;
    for(int i=0; i<names.size(); i++)
    {
        if(i%5 == 0)
        {
            v.push_back(names[i]);
        }
    }
    return v;
}

File: 프로그래머스/0/181886. 5명씩/5명씩.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public string[] solution(string[] names) 
    {
        List<string> nameList = new List<string>();
        
        for (int i=0; i<names.Length; i++) {
            if (i%5 == 0) {
                nameList.Add(names[i]);
            }
        }
        return nameList.ToArray();
    }
}